release/package.json

```diff
@@ -921,6 +925,10 @@
           "command": "fsharp.explorer.moveDown",
           "when": "false"
         },
+        {
+          "command": "fsharp.explorer.renameFile",
+          "when": "false"
```

Comment:
should this have a different condition? or is it only intended to be triggered from the explorer?